src/components/ProjectSearch.js

```diff
@@ -329,7 +322,10 @@ export class ProjectSearch extends Component<Props, State> {
         onBlur={() => this.setState({ inputFocused: false })}
         onKeyDown={this.onKeyDown}
         onHistoryScroll={this.onHistoryScroll}
-        handleClose={this.props.closeProjectSearch}
+        handleClose={
+          // TODO - This function doesn't quite match the signature.
+          (this.props.closeProjectSearch: any)
```

Comment:
The type signature doesn't match the usage. This needs the `SearchInput` component's types to be more permissive to what you give it I think.